datahub-web-react/src/app/search/SearchBar.tsx

```diff
@@ -161,6 +161,7 @@ interface Props {
     autoCompleteStyle?: React.CSSProperties;
     entityRegistry: EntityRegistry;
     fixAutoComplete?: boolean;
+    setIsInSearchBar?: (isInSearchBar: boolean) => void;
```

Comment:
lets call this `setIsSearchBarFocused` to be specific?

Reply:
++